pkgs/applications/version-management/forgejo/default.nix

```
@@ -66,46 +70,61 @@ buildGoModule rec {
       --prefix PATH : ${lib.makeBinPath [ bash git gzip openssh ]}
   '';
 
-  passthru.tests = nixosTests.forgejo;
-
-  passthru.updateScript = lib.getExe (writeShellApplication {
-    name = "update-forgejo";
-    runtimeInputs = [
-      common-updater-scripts
-      coreutils
-      curl
-      jq
-      nix
-    ];
-    text = ''
-      releases=$(curl "https://codeberg.org/api/v1/repos/forgejo/forgejo/releases?draft=false&pre-release=false&limit=1" \
-        --silent \
-        --header "accept: application/json")
-
-      stable=$(jq '.[0]
-        | .tag_name[1:] as $version
-        | ("forgejo-src-\($version).tar.gz") as $filename
-        | { $version, html_url } + (.assets | map(select(.name | startswith($filename)) | {(.name | split(".") | last): .browser_download_url}) | add)' \
-        <<< "$releases")
-
-      archive_url=$(jq -r .gz <<< "$stable")
-      checksum_url=$(jq -r .sha256 <<< "$stable")
-      release_url=$(jq -r .html_url <<< "$stable")
-      version=$(jq -r .version <<< "$stable")
-
-      if [[ "${version}" = "$version" ]]; then
-        echo "No new version found (already at $version)"
-        exit 0
-      fi
-
-      echo "Release: $release_url"
-
-      sha256=$(curl "$checksum_url" --silent | cut --delimiter " " --fields 1)
-      sri_hash=$(nix hash to-sri --type sha256 "$sha256")
-
-      update-source-version "${pname}" "$version" "$sri_hash" "$archive_url"
+  passthru = {
+    data-compressed = runCommand "data-compressed" {
+      nativeBuildInputs = [ brotli xorg.lndir ];
+    } ''
+      mkdir $out
+      lndir ${forgejo.data}/ $out/
+
+      # Create static gzip and brotli files
+      find -L $out -type f -regextype posix-extended -iregex '.*\.(css|html|js|svg|ttf|txt)' | while read file; do
+        gzip --best --keep --force $file
+        brotli --best --keep $file
```

```suggestion
        brotli --best --keep --no-copy-stat $file
```

To get rid of all the
```
setting group failed for [/nix/store/jkrq0xdlw50ga14380v3qjlj0lm0rq5x-data-compressed/public/css/dropzone.0142b04d.css.br]: Invalid argument
setting user failed for [/nix/store/jkrq0xdlw50ga14380v3qjlj0lm0rq5x-data-compressed/public/css/dropzone.0142b04d.css.br]: Invalid argument
```
warnings

---

Thanks, fixed.